### PR TITLE
Dockerize (revised)

### PR DESCRIPTION
**Pull request by emkookmer**
I've integrated jangottweiss's Docker implementatie and improved upon it.

I've also added a couple of other things which I found useful for my own.
Please review them.

I'm recreating this pull request because in the past 7 months there have been some major changes, due to this I could not rebase the commits for #78 , thus i had to recreate ever step.

### CODE REVIEW

**Comment by Sennevds** [comment]
Sorry I just merged MaxVram's PR and now you have some merge conflicts. Could you resolve these and then I will check the PR

**Comment by emkookmer** [comment]
Hi @Sennevds ,
The Merge conflicts are resolved

**Comment by whalecoiner** [comment]
This would be very useful to have! Would love to see it merged!

File: src/system_sensors.py [hunk 1]
```diff
@@ -197,6 +222,8 @@ def on_message(client, userdata, message):
 
     devicename = settings['devicename'].replace(' ', '').lower()
     deviceNameDisplay = settings['devicename']
+    deviceModel = get_host_model()
+    deviceManufacturer = "RPI Foundation" if "rasp" in OS_DATA["ID"] else OS_DATA['Name']
```
Comment (mannkind) [inline src/system_sensors.py:226]: Should `'Name'` be `’NAME’`

It’s uppercase on my RPi and x86 boxes.

**Comment by svh1985** [comment]
@Sennevds can you please check if this can be merged? Would be awesome to run in in docker!

Thanks!

File: src/system_sensors.py (hunk 1 above)
Comment (emkookmer) [reply src/system_sensors.py]: Nice catch 👍

**Comment by boschr** [comment]
Another try 🙏  @Sennevds do you see any way to get this merged? I highly prefer to use docker. Thanks in advance 😄

**Comment by Sennevds** [comment]
Sorry for the late reply and merge. I've resolved the issues it should be merged now

**Pull request merged**